### PR TITLE
Add new organization elvanto

### PR DESCRIPTION
**Pull request by bmcclenahan**

### CODE REVIEW

**Comment by bmcclenahan** [comment]
I've signed it

**Comment by bmcclenahan** [comment]
I've signed a CLA

File: organizations/elvanto.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "ELVANTO",
+  "name": "elvanto",
```
Comment (pfrisella) [inline organizations/elvanto.json]:
```suggestion
  "name": "Elvanto",
```

**Comment by bmcclenahan** [comment]
the suggested changes have been made
the author consents
please progress the pull request

**Comment by bmcclenahan** [comment]
the suggested changes have been made
the author consents
please progress the pull request

**Pull request merged**